enterprise/frontend/src/embedding-sdk-ee/tests/auth-flow/jwt.unit.spec.tsx

```diff
@@ -29,10 +29,25 @@ const setup = ({
   locale,
 }: Pick<MetabaseProviderProps, "authConfig" | "locale">) => {
   setupMockJwtEndpoints();
+
+  const getFirstSsoDiscoveryCall = () => {
+    // This is `/auth/sso`/`auth/sso?preferred_method=...`
```

Comment:
nit using `/` to separate the two urls are pretty confusing given there are `/` in the URLs themselves. 


```suggestion
    // This is `/auth/sso` or `auth/sso?preferred_method=...`
```